hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
I ran the command the script uses manually on this file:
```
git log --date=short --format=format:%ad hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp
```
and according to the git log 2016 is correct.  Whatever history existed for this file before 2016 isn't in the repo anymore.  Perhaps it was lost when the code got uploaded to github.  My guess is this file was created in 2015 originally but the first update in github was 2016.  Since we don't have the full history, I think 2016 is the best we can do.  Plus the copyright headers are probably most needed/relevant when hootenanny was put in github as open source.